internal/core/metadata/uom/bootstrap.go

```diff
@@ -0,0 +1,61 @@
+//
+// Copyright (C) 2022 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package uom
+
+import (
+	"context"
+	"io/ioutil"
+	"sync"
+
+	bootstrapContainer "github.com/edgexfoundry/go-mod-bootstrap/v2/bootstrap/container"
+	"github.com/edgexfoundry/go-mod-bootstrap/v2/bootstrap/startup"
+	"github.com/edgexfoundry/go-mod-bootstrap/v2/di"
+	"github.com/pelletier/go-toml"
+
+	"github.com/edgexfoundry/edgex-go/internal/core/metadata/container"
+)
+
+func BootstrapHandler(_ context.Context, _ *sync.WaitGroup, _ startup.Timer, dic *di.Container) bool {
+	lc := bootstrapContainer.LoggingClientFrom(dic.Get)
+	config := container.ConfigurationFrom(dic.Get)
+
+	var uomImpl *UnitsOfMeasureImpl
+	filepath := config.UoM.UoMFile
+	// backward compatability for using older 2.x configuration
+	// TODO: Remove in EdgeX 3.0
+	if filepath == "" {
+		dic.Update(di.ServiceConstructorMap{
+			container.UnitsOfMeasureInterfaceName: func(get di.Get) interface{} {
+				return uomImpl
+			},
+		})
+
+		lc.Warn("UoM.UoMFile field not set in configuration file, unit of measure validation will always fail")
```

Comment:
Why `unit of measure validation will always fail` if `uomImpl` is empty? 
```suggestion
		lc.Warn("UoM.UoMFile field not set in configuration file, unit of measure validation is disabled")
```

Reply:
> Should check for empty string and create empty UnitsOfMeasureImpl when it is empty. Validation will also be missing and default to false.

Based on previous review comment, I think default to false means fail the validation .
If  we want to disable this should we return true instead ?

Reply:
Correct, that is why I commented on the actual validation to return true if the UOM list is empty.